plugins/outputs/bigquery/bigquery.go

```diff
@@ -138,29 +171,63 @@ func newValuesSaver(m telegraf.Metric) *bigquery.ValuesSaver {
 	}
 }
 
+func (s *BigQuery) newCompactValuesSaver(m telegraf.Metric) *bigquery.ValuesSaver {
+	tags, err := json.Marshal(m.Tags())
+	if err != nil {
+		s.Log.Warnf("serializing tags: %v", err)
+	}
+
+	fields, err := json.Marshal(m.Fields())
+	if err != nil {
+		s.Log.Warnf("serializing fields: %v", err)
+	}
+
```

Comment:
Does it really make sense to send partial data?

Reply:
Yeah I would incline to log the error and return `nil` to drop the metrics instead of retrying... But I'm OK with partial data as long as this makes sense (and a comment describing why it makes sense in the code). :-)